ui-tests/src/test/java/org/openmrs/reference/TransferToWardServiceTest.java

```diff
@@ -34,19 +36,21 @@ public void createTestData() throws Exception {
         String visit = new TestData.TestVisit(testPatient.uuid, visitType, getLocationUuid(homePage)).create();
     }
 
-    @Ignore("TODO fix, inpatient_ward doesn't exisit on the page, fix workflow")
     @Test
-    public void transferToWardServiceTest() throws Exception {
+    @Category(BuildTests.class)
+    public void transferToWardServiceTest() {
 
         ClinicianFacingPatientDashboardPage patientDashboardPage = homePage.goToActiveVisitPatient();
 
+        patientDashboardPage.clickOnLocation();
         patientDashboardPage.goToAdmitToInpatient().confirm(INPATIENT_WARD);
         patientDashboardPage.waitForPage();
 
+        patientDashboardPage.clickOnLocation();
         patientDashboardPage.goToTransferToWardServicePage().confirm(ISOLATION_WARD);
         patientDashboardPage.waitForPage();
 
-        assertTrue(patientDashboardPage.containsText("Outpatient"));
+        assertTrue(patientDashboardPage.containsText(ISOLATION_WARD));
```

Comment:
Since you catered for ISOLATION_WARD  element  for the above lines, why adding it on line 53 again